sysl2/sysl/tests/go.gen.sysl

```diff
@@ -0,0 +1,245 @@
+Go:
```

Comment:
I would like this file to be as minimal as possible. Only keep what is relevant to the test case.